ci-operator/config/monstorak/monstorak-operator/monstorak-monstorak-operator-master.yaml

```diff
@@ -0,0 +1,56 @@
+canonical_go_repository: github.com/monstorak/monstorak-operator
+
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: '7'
+  operator-registry:
+    name: "4.0"
+    namespace: ocp
+    tag: operator-registry
+
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      memory: 250Mi
+      cpu: 100m
+
+images:
+- from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/monstorak/monstorak-operator/build/_output/bin/monstorak-operator 
+  dockerfile_path: openshift-ci/Dockerfile.deploy
+  to: monstorak-operator
+
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
+  name: monstorak-0.0.1
```

Comment:
this imagestream doesn't exist. Is this intentional? You will need also to add an imagestream creation as well.

Reply:
@droslean I think ci-operator creates target ImageStreams if they do not exist on promoting.

https://github.com/openshift/ci-tools/blob/master/pkg/steps/release/promote.go#L83